crates/resolver/src/ingot.rs

```diff
@@ -47,16 +47,23 @@ impl ResolutionHandler<FilesResolver> for BasicIngotNodeHandler {
             return config
                 .dependencies
                 .into_iter()
-                .map(|dependency| match dependency.description {
-                    DependencyDescription::Path(path) => (
-                        ingot_path.join(path).canonicalize_utf8().unwrap(),
```

Comment:
Seems like the resolver was handling relative paths quite well before.  If `Dependency`/`DependencyDescription` can only store a `Url` this gets trickier.

Reply:
I don't think the dependency struct should hold onto a url, unless it is a git repo or something. it should just be a simple representation of what is in the config file. it's pretty easy for a function in common::ingot to map the path `DependencyDescription` to a file url given the parent ingot.